src/cascadia/TerminalSettingsEditor/Appearances.cpp

```diff
@@ -588,29 +586,49 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         return str == end || errnoRef == ERANGE ? NAN : value;
     }
 
-    void AppearanceViewModel::LineHeight(const double value)
+    double AppearanceViewModel::LineHeight() const
     {
-        std::wstring str;
+        const auto cellHeight = _appearance.SourceProfile().FontInfo().CellHeight();
+        return _extractCellSizeValue(cellHeight);
+    }
 
-        if (value >= 0.1 && value <= 10.0)
-        {
-            str = fmt::format(FMT_STRING(L"{:.6g}"), value);
-        }
+    double AppearanceViewModel::CellWidth() const
+    {
+        const auto cellWidth = _appearance.SourceProfile().FontInfo().CellHeight();
+        return _extractCellSizeValue(cellWidth);
+    }
 
-        const auto fontInfo = _appearance.SourceProfile().FontInfo();
+#define CELL_SIZE_SETTER(modelName, viewModelName)                 \
+    std::wstring str;                                              \
+                                                                   \
+    if (value >= 0.1 && value <= 10.0)                             \
+    {                                                              \
+        str = fmt::format(FMT_STRING(L"{:.6g}"), value);           \
```

Comment:
hey @Lhecker why is this `STRING` and not `COMPILE`?

Reply:
I blame Copilot haha. Changed to `COMPILE`

Reply:
ಠ_ಠ

Reply:
Sorry, didn't see this. `FMT_STRING` used to be for doing compile-time checks but runtime formatting (compact assembly). Nowadays compile-time checks are always done.